apps/portals/src/configurations/challengeportal/synapseConfigs/challenges.ts

```diff
@@ -278,6 +278,12 @@ export const challengeDetailsPageConfig: DetailsPageProps = {
 }
 
 export const challengeDetailsLandingPage: SynapseConfig[] = [
+  // Register For Challenge component is here, and the wrapper positions it properly in the header card
+  {
+    name: 'ChallengeDetailPageWrapper',
```

Comment:
Wrapper pulls challenge project ID from the search param, and does some hacky things to make sure the registration button shows up in a relatively nice position on screen